enhancements/ingress/ingress-dashboard.md

```diff
@@ -0,0 +1,199 @@
+---
+title: ingress-dashboard
+authors:
+  - "@jotak"
+reviewers:
+  - "@Miciah"
+  - "@candita"
+  - "@OlivierCazade"
+approvers:
+  - "@Miciah"
+  - "@candita"
+api-approvers:
+  - "@deads2k"
+creation-date: 2023-06-28
+last-updated: 2023-06-28
+tracking-link:
+  - "https://issues.redhat.com/browse/OCPSTRAT-139"
+  - "https://issues.redhat.com/browse/NETOBSERV-1052"
+see-also:
+replaces:
+superseded-by:
+---
+
+# Ingress dashboard creation for the OpenShift Console
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`.
+- [ ] Design details are appropriately documented from clear requirements.
+- [ ] Test plan is defined.
+- [ ] graduation criteria for dev preview, tech preview, GA
+- [ ] User-Facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/).
+
+## Summary
+
+Add a new dashboard in the OpenShift Console (in menu “Observe” > “Dashboards”), dedicated to metrics related to Ingress.
+
+Ingress components, such as HAProxy, already provide some metrics that are exposed and collected by Prometheus / Cluster Monitoring. Administrators should be able to get a consolidated view, using a subset of these metrics, to get a quick overview of the cluster state. This enhancement proposal is part of a wider initiative to improve the observability of networking components (cf https://issues.redhat.com/browse/OCPSTRAT-139).
+
+## Motivation
+
+While ingress related metrics already exist and are accessible in the OpenShift Console (via the menu “Observe” > Metrics”), there is no consolidated view presenting a summary of them. There are existing dashboards in other areas (such as etcd, compute resources, etc.), but networking today is less represented there, despite its importance for monitoring and troubleshooting.
```

Comment:
Can you begin listing what general things are important for monitoring and troubleshooting?  This will help us pick which of the existing metrics we have that we can add to the dashboard.

Reply:
On HAProxy, metrics such as error rates or latencies are certainly important to monitor; I've updated the EP to insist on these ones.
PMs have asked for cluster level statistics on number of routes and shards to use, so this is also mentioned here